### PR TITLE
chore(framework): add sx to toolbox content

### PR DESCRIPTION
**Pull request by GustavHF73**
Toolbox content accepts sx as prop in order to be able to override styling. Changed children wrapper in toolbox from box to vstack to be able to use flex grow for children

### CODE REVIEW

File: framework/lib/components/toolbox/toolbox.tsx [hunk 1]
```diff
@@ -2,6 +2,7 @@ import React from 'react'
 import { useMultiStyleConfig, useToken } from '@chakra-ui/system'
 import { coreZIndex } from '@northlight/tokens'
 import { FocusScope } from '@react-aria/focus'
+import { VStack } from '@chakra-ui/react'
```
Comment (SebCodesTheWeb) [inline framework/lib/components/toolbox/toolbox.tsx:5]: Screenshot 2024-08-02 at 11 14 45
it should be imported from lib/stack

File: framework/lib/components/toolbox/toolbox.tsx [hunk 2]
```diff
@@ -117,9 +118,9 @@ export const Toolbox = ({
           >
             { isResizable && <ResizeHandle { ...resizeProps } /> }
             <FocusScope autoFocus={ autoFocus }>
-              <Box width="full" height="full">
+              <VStack width="full" height="full">
```
Comment (SebCodesTheWeb) [inline framework/lib/components/toolbox/toolbox.tsx:121]: also why this change

Comment (GustavHF73) [reply framework/lib/components/toolbox/toolbox.tsx:121]: This is because we want the display flex and column on the container, in order to make the flex-grow prop on the children work

**Review by SebCodesTheWeb — APPROVED** [review]
